### DISCUSSION TITLE
Overdrive audiobooks

### DISCUSSION THREAD

**Discussion by 45Bulletproof** [enzo1982/freac#529] [Your Ideas]
Just wanted to let people know that freac does a beautiful job of breaking Overdrive audiobooks (think library) into chapters.  Freac is one of my better discoveries for the past year.

**Comment by Lee-Carre** [comment]
Good to know, thankyou @45Bulletproof.

How do Overdrive audiobooks come packaged? I've never heard of them. I presume as a single large file. Else, if on CDDA, how are the tracks organised?

The CDDA audiobooks available at my local library seem to all be from Clipper, which has the nasty habit of having a track-break every 3 minutes or so (seemingly so that people can easily resume playback, later, with moderate resolution, without lots of seeking on non-computer CD players). Not track-per-chapter as one would hope/like. Figuring out a way to remedy that, is still on my to-do list.
Hence asking, incase your situation is similar.

**Reply by 45Bulletproof** [reply]
The Overdrive books (from or rather through the library) come as a series of mp3 files.  They are bunches of chapters grouped into what seems like about 1 hour intervals that form one of the mp3 files.  It is just a matter of dropping them into freac as a group in the order they are supposed to be in.  And (I'm thinking it's in the metadata), they just get turned into chapters if you want individual files or a large m4b file with chapters embedded in it.  Something I discovered by accident, as my knowledge of the nuts and bolts of this is minimal.
